src/PokemonContainer.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import PokeList from './PokeList.js';
+
+class PokemonContainer extends Component {
+    state = { 
+        data: [],
+        loading: true,
+        query: null,
```

Comment:
Remember, the data you initialize into state to start with should match the data TYPE that you plan to store in state. In this case, query should be an empty string since you will be storing a string value in it.